File: core_library/src/main/java/core/view/CoreComponents.java
package core.view;

import android.content.Context;
import android.view.View;
import android.widget.SeekBar;

public class CoreComponents {

    public static SeekBar createSeekBarRadius(Context context) {
        SeekBar seekBar = new SeekBar(context);
        seekBar.setMax(100);
        seekBar.setVisibility(View.INVISIBLE);
        return seekBar;
    }
}

File: core_library_kt/src/main/java/core_kt/view/CoreComponents.kt
package core_kt.view

import android.content.Context
import android.view.View
import android.widget.SeekBar

fun createSeekBarRadius(context: Context): SeekBar {
    val seekBar = SeekBar(context)
    seekBar.max = 100
    seekBar.visibility = View.INVISIBLE
    return seekBar
}